sql/payday.sql

```diff
@@ -186,13 +186,14 @@ CREATE OR REPLACE FUNCTION process_payment_instruction() RETURNS trigger AS $$
               FROM payday_participants p
              WHERE username = NEW.participant
         );
+
         IF (NEW.amount + NEW.due <= participant.new_balance OR participant.card_hold_ok) THEN
             EXECUTE pay(NEW.participant, NEW.team, NEW.amount + NEW.due, 'to-team');
             RETURN NEW;
-        ELSE
+        ELSIF NEW.amount + NEW.due <= (SELECT minimum_charge FROM settings) THEN
```

Comment:
> We don't want $260 surprises!
> but isn't it kinda weird to use MINIMUM_CHARGE as a maximum now?
This PR solves the $260 surprise problem, but as you said - the semantics don't seem right..